static/templates/message_reaction.handlebars

```diff
@@ -0,0 +1,4 @@
+<div class="message_reaction" title="{{this.title}}" data-emoji-name="{{this.emoji_name}}">
+  <div class="emoji emoji-{{this.emoji_name}}" />
+  <div class="message_reaction_count">{{this.count}}</div>
+</div>
```

Comment:
message_reaction.handlebars is used in add_reaction() (in reactions.js) to append new reactions to the message reactions div.

message_reactions.handlebars is used to generate the message reactions div on first load. This could be inlined into single_message.handlebars, but that file is big enough :)